syntax = "proto3";

package echo;

message EchoRequest {
  string text = 1;
}

message EchoResponse {
  string result = 1;
}

service EchoService {
  rpc echo(EchoRequest) returns (EchoResponse);
}
